src/vs/workbench/workbench.desktop.main.ts

```diff
@@ -53,7 +53,7 @@ import 'vs/workbench/services/userDataSync/electron-browser/userDataSyncStoreMan
 import 'vs/workbench/services/userDataSync/electron-browser/userDataAutoSyncService';
 import 'vs/workbench/services/sharedProcess/electron-browser/sharedProcessService';
 import 'vs/workbench/services/localizations/electron-browser/localizationsService';
-import 'vs/workbench/services/experiment/electron-browser/experimentService';
+import 'vs/workbench/services/experiment/common/experimentService';
```

Comment:
Instead of importing this both from web and desktop, why not move it into `workbench.common.main.ts` which is shared between the two?